oauth2-authorization-server/src/main/java/org/springframework/security/oauth2/core/OAuth2AuthorizationServerMetadataClaimNames.java

```diff
@@ -51,6 +51,11 @@
 	 */
 	String JWKS_URI = "jwks_uri";
 
+	/**
+	 * {@code userinfo_endpoint} - the {@code URL} of the OAuth 2.0 UserInfo Endpoint
+	 */
+	String USER_INFO_ENDPOINT = "userinfo_endpoint";
```

Comment:
Related to supporting only `oidc-configuration`, please move this to `OidcProviderMetadataClaimNames`.